### DISCUSSION TITLE
Multi-touch support

### DISCUSSION THREAD

**Discussion by x-iso** [rustdesk/rustdesk#2399] [General]
If there's one thing that missing in most apps like this, it's relay of multi-touch input from touchscreen capable client to the host. Well, to be precise Teamviewer actually does have this, but it became quite unreliable otherwise. So, would it be possible to implement? I imagine that on the host side app should use virtual touch input driver for this to work.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/3744
